### PR TITLE
Add the PIMA pv data manager

### PR DESCRIPTION
**Pull request by gregvonkuster**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: data_managers/data_manager_pima_pv_downloader/test-data/pima_pv.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"data_tables": {"pima_pv": {"name": "2
```
Comment (bgruening) [inline data_managers/data_manager_pima_pv_downloader/test-data/pima_pv.json:1]: that looks wrong :)

File: data_managers/data_manager_pima_pv_downloader/test-data/pima_pv.loc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# This is a sample file distributed with Galaxy that enables tools
```
Comment (bgruening) [inline data_managers/data_manager_pima_pv_downloader/test-data/pima_pv.loc:1]: shouldn't this file be called prima_pv.loc.test or so?

File: data_managers/data_manager_pima_pv_downloader/test-data/pima_pv.json (hunk 1 above)
Comment (gregvonkuster) [reply data_managers/data_manager_pima_pv_downloader/test-data/pima_pv.json]: Fixed the typo - tanks! :)

File: data_managers/data_manager_pima_pv_downloader/test-data/pima_pv.loc (hunk 2 above)
Comment (gregvonkuster) [reply data_managers/data_manager_pima_pv_downloader/test-data/pima_pv.loc:1]: Te test-data dirs just have .loc files (e.g., https://github.com/galaxyproject/tools-iuc/tree/master/data_managers/data_manager_bowtie2_index_builder/test-data).  The .test files for DMs are only in the root dir - https://github.com/galaxyproject/tools-iuc/blob/master/data_managers/data_manager_bowtie2_index_builder/tool_data_table_conf.xml.test

**Comment by gregvonkuster** [comment]
Is the failing R lint relevant?  This tool isn;'t written in R.
```
[Lint R scripts (ubuntu-20.04, 4.0.1)](https://github.com/galaxyproject/tools-iuc/pull/4849#logs)
failed 3 hours ago in 1s
 
Set up job
...
Error: Unable to resolve action `r-lib/actions@master`, unable to find version `master`
```

**Comment by nsoranzo** [comment]
> Is the failing R lint relevant? This tool isn;'t written in R.
> 
> ```
> [Lint R scripts (ubuntu-20.04, 4.0.1)](https://github.com/galaxyproject/tools-iuc/pull/4849#logs)
> failed 3 hours ago in 1s
>  
> Set up job
> ...
> Error: Unable to resolve action `r-lib/actions@master`, unable to find version `master`
> ```

This was caused by a change in the GitHub action we use to lint R scripts, it's now fixed by https://github.com/galaxyproject/tools-iuc/pull/4850 , sorry for the inconvenience. Can you rebase or merge the master branch?

**Pull request closed without merging**